portfolio/README.md

```diff
@@ -5,4 +5,4 @@ command:
 
 ```bash
 mvn package appengine:run
-```
\ No newline at end of file
+```
```

Comment:
nit - revert this change since it's unrelated to the PR.

Reply:
thanks! updated!